src/transaction/declare.rs

```diff
@@ -508,184 +494,6 @@ mod tests {
         );
     }
 
-    #[test]
```

Comment:
The deleted tests use the (now removed) verify_version function. I think we should have an equivalent test that checks the same. Just to avoid losing tests that check known conditions for failure

Reply:
Sure, Ill add a couple tests